### PR TITLE
a few bug fixes applied.

### PR DESCRIPTION
**Pull request by LottieRugg**

### CODE REVIEW

**Comment by LottieRugg** [comment]
Still buggin on the nav bar and the "let's work together" button on on very small screen sizes. 
...and the submit links and nav links don't work...yet.

File: stylesheet.css [hunk 1]
```diff
@@ -118,89 +103,79 @@ button{
...
-    color:#6a6a6a;
+    color: #6a6a6a;
+}
+.submit_button a{
+    text-decoration: none;
 }
-
 /* Footer bar styling */
 footer{
     width: 100%;
```
Comment (haefnir) [inline stylesheet.css]: width not needed, default is 100%

File: stylesheet.css [hunk 2]
```diff
@@ -118,89 +103,79 @@ button{
...
-    height:70px;
+    height: 70px;
     background-color: #fae3e5;
-    color:#214794;
+    color: #214794;
```
Comment (haefnir) [inline stylesheet.css]: `footer a{` is more specific so this does nothing

File: stylesheet.css [hunk 3]
```diff
@@ -118,89 +103,79 @@ button{
...
 @media only screen and (max-width:1266px){
     .my_experience{
         display: grid;
         grid-template-columns: 100%;
     }
-    /* a .cv_icon{
-        margin:auto;
-    } */
     .my_experience{
         background-color: #fae3e5;
```
Comment (haefnir) [inline stylesheet.css]: not needed, already defined on line 108

File: stylesheet.css [hunk 4]
```diff
@@ -118,89 +103,79 @@ button{
...
-
 .experience_text{
     padding: 150px;
 }
-
 /* Contact form styling */
-
 .contact_form_section{
     background-image: url("Images/Background.jpg");
     width: 100%;
```
Comment (haefnir) [inline stylesheet.css]: not needed, default is 100%

File: stylesheet.css [hunk 5]
```diff
@@ -5,94 +5,79 @@
...
 nav i{
-    padding: 0  5px 0 0;
+    padding-right: 5px;
 }
-
 /* Header styling  */
-
 header{
     background-image: url("Images/Background.jpg");
     width: 100%;
```
Comment (haefnir) [inline stylesheet.css]: not needed, default is 100%

File: stylesheet.css [hunk 6]
```diff
@@ -5,94 +5,79 @@
...
 
 nav{
-    display:flex;
-    flex-direction:row;
-    justify-content:space-between;
-    align-items:center;
-    width:100%;
-    padding: 20px 0px 20px 0px;
+    display: flex;
+    flex-direction: row;
```
Comment (haefnir) [inline stylesheet.css:17]: not needed, default value

File: stylesheet.css [hunk 7]
```diff
@@ -5,94 +5,79 @@
...
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
```
Comment (haefnir) [inline stylesheet.css]: default value

File: stylesheet.css [hunk 8]
```diff
@@ -5,94 +5,79 @@
...
     margin-left: auto;
     margin-right: auto;
-    margin-top:30px;
+    margin-top: 30px;
     font-size: 20px;
     border-radius: 45px;
     border-style: none;
     background-color: #FFFFFF;
     position: relative;
     z-index: 2;
```
Comment (haefnir) [inline stylesheet.css:75]: this doesn't seem to be doing anything

File: stylesheet.css [hunk 9]
```diff
@@ -118,89 +103,79 @@ button{
...
+    height: 70px;
     background-color: #fae3e5;
-    color:#214794;
+    color: #214794;
     padding: 20px;
 }
 footer a{
-    color:#214794;
-    padding:5px;
+    color: #214794;
```
Comment (haefnir) [inline stylesheet.css]: all `<a>` appear to have this styling. Could remove and make general rule
```
a {
    color:#214794;
}
```

File: stylesheet.css [hunk 10]
```diff
@@ -118,89 +103,79 @@ button{
...
     color: #e2431b;
 }
-
 .email_link{
     float: right;
 }
-
 @media only screen and (max-width:1266px){
     .my_experience{
         display: grid;
```
Comment (haefnir) [inline stylesheet.css]: already set to `display: grid` 
can delete this

File: stylesheet.css [hunk 11]
```diff
@@ -118,89 +103,79 @@ button{
...
         grid-template-columns: 100%;
     }
```
Comment (haefnir) [inline stylesheet.css]: two rulesets for .my-experience in this block. Consolidate changes and delete one.

File: stylesheet.css [hunk 12]
```diff
@@ -258,33 +226,32 @@ footer a:hover{
...
     .project_image img{
-        height:auto;
+        height: auto;
     }
     .project_text{
-        margin:0;
+        margin: 0;
     }
     .my_work{
         display: grid;
```
Comment (haefnir) [inline stylesheet.css]: already set to `display:grid`

File: stylesheet.css [hunk 13]
```diff
@@ -258,33 +226,32 @@ footer a:hover{
...
         grid-template-columns: 100%;
-        margin:0;
-        padding:10px;
+        margin: 0;
+        padding: 10px;
     }
     .my_work_two{
         display: grid;
```
Comment (haefnir) [inline stylesheet.css]: already set to `display:grid`

File: stylesheet.css [hunk 14]
```diff
@@ -118,89 +103,79 @@ button{
...
     background-image: url("Images/Background.jpg");
     width: 100%;
     height: 850px;
-    padding-top:100px;
-    
+    padding-top: 100px;
 }
 .contact_form{
     background-color: #FFFFFF;
     padding: 60px 40px;
```
Comment (haefnir) [inline stylesheet.css]: this is being overriden by next padding instruction

File: stylesheet.css [hunk 15]
```diff
@@ -105,11 +90,11 @@ button{
 }
 .project_image img{
    height: 400px;
-   width:auto;
+   width: auto;
```
Comment (haefnir) [inline stylesheet.css]: default value

File: stylesheet.css [hunk 16]
```diff
@@ -258,33 +226,32 @@ footer a:hover{
...
     .project_text{
-        margin:0;
+        margin: 0;
     }
     .my_work{
         display: grid;
         grid-template-columns: 100%;
-        margin:0;
-        padding:10px;
+        margin: 0;
```
Comment (haefnir) [inline stylesheet.css:255]: This does nothing

File: stylesheet.css [hunk 17]
```diff
@@ -213,31 +188,24 @@ footer a:hover{
...
+        display: grid;
         grid-template-columns: 100%;
     }
-    /* nav a{
-        width:100%;
-    } */
-  
-
     .intro{
         display: grid;
```
Comment (haefnir) [inline stylesheet.css]: already defined earlier

File: stylesheet.css [hunk 18]
```diff
@@ -213,31 +188,24 @@ footer a:hover{
...
         grid-template-columns: 100%;
-        /* margin: 20px; */
+      
         margin:0;
```
Comment (haefnir) [inline stylesheet.css:212]: does nothing

File: stylesheet.css [hunk 19]
```diff
@@ -213,31 +188,24 @@ footer a:hover{
...
         padding:0;
         background-color: #fae3e5;
     }
     .intro_item.intro_text{
-        background-color:#fae3e5;
+        background-color: #fae3e5;
         padding: 10px ;
         width: 100%;
```
Comment (haefnir) [inline stylesheet.css:219]: does nothing

File: stylesheet.css [hunk 20]
```diff
@@ -258,33 +226,32 @@ footer a:hover{
     }
     img{
         max-width: 100%;
-        height:auto;
+        height: auto;
```
Comment (haefnir) [inline stylesheet.css:237]: doesn't seem to do anything

**Review by haefnir — CHANGES_REQUESTED** [review]
line 147 and 150 should probably `target="_blank"` 

add normalize.css to your project

let me know when this is pushed and will check again :)

File: stylesheet.css (hunk 16 above)
Comment (LottieRugg) [reply stylesheet.css:255]: I'm going to keep this as it is needed to style for small screen sizes.

File: stylesheet.css (hunk 19 above)
Comment (LottieRugg) [reply stylesheet.css:219]: on v small screen sizes removes the background image

File: stylesheet.css (hunk 20 above)
Comment (LottieRugg) [reply stylesheet.css:237]: stops images stretching out of proportion on small screens

**Comment by LottieRugg** [comment]
Thank you for your very thorough review Seb!!!👑

File: index.html [hunk 21]
```diff
@@ -4,12 +4,12 @@
...
+    <link href="normalize.css" rel="stylesheet" type="text/css">
     <script src="https://kit.fontawesome.com/40c7107b4d.js" crossorigin="anonymous"></script>
     <link rel="stylesheet" href="stylesheet.css" type="text/css">
     <link rel="preconnect" href="https://fonts.googleapis.com">
-<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
-
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
     <title>Portfolio</title>
```
Comment (haefnir) [inline index.html]: Would be a good idea to have your name in the title!

File: index.html [hunk 22]
```diff
@@ -34,65 +34,74 @@
...
+                View live project
+                <i class="fa-solid fa-arrow-right"></i>
+            </button>
+        </a>
+        <a href="#">
+            <button class="project_button">  
+            View on GitHub
+            <i class="fa-solid fa-arrow-right"></i>
+            </button>
+        </a>
```
Comment (haefnir) [inline index.html:74]: inconsistent indentation

File: stylesheet.css [hunk 23]
```diff
@@ -105,144 +99,141 @@ button{
...
 }
 
+a{
+    color:#214794;
+    text-decoration: none;
+}
+
 @media only screen and (max-width:1266px){
     .my_experience{
         display: grid;
```
Comment (haefnir) [inline stylesheet.css]: this repeats. delete. also defined earlier in the style sheet

File: stylesheet.css [hunk 24]
```diff
@@ -105,144 +99,141 @@ button{
...
-    }
-    /* a .cv_icon{
-        margin:auto;
-    } */
-    .my_experience{
         background-color: #fae3e5;
         background-size: cover;
         background-position: center;
         margin: 0px;
         display: grid;
```
Comment (haefnir) [inline stylesheet.css]: defined earlier in the stylesheet, not needed

**Review by haefnir — CHANGES_REQUESTED** [review]
scaling issue around viewport width 990px. Fix then push back for review :)

File: stylesheet.css [hunk 25]
```diff
@@ -105,186 +99,207 @@ button{
...
+    nav a{
+        display: grid;
+		grid-template-columns: 100%;
+	}
+}
+
+@media only screen and (max-width:505px){
+    .cv_image{
+        width:80%;
+        /* place-self:center; */
```
Comment (haefnir) [inline stylesheet.css]: delete this if its not going to feature

File: stylesheet.css [hunk 26]
```diff
@@ -105,186 +99,207 @@ button{
...
     }
-    
     .contact_form{
         background-color:#FFFFFF;
-        padding:0;
+        padding: 0;
         padding: 10px;
-        margin: auto;
-        width:100%
+        /* margin: auto; */
```
Comment (haefnir) [inline stylesheet.css]: delete this if its not going to feature

File: stylesheet.css [hunk 27]
```diff
@@ -105,186 +99,207 @@ button{
...
     }
     .experience_text{
         padding: 10px;
     }
-    .my_experience a{
-        margin:auto;
-    }
+    /* .my_experience a{
+        margin: auto;
+    } */
```
Comment (haefnir) [inline stylesheet.css]: delete if not going to feature

File: stylesheet.css [hunk 28]
```diff
@@ -105,186 +99,207 @@ button{
...
 textarea::placeholder{
-    color:#6a6a6a;
+    color: #6a6a6a;
+}
+.submit_button a{
+    text-decoration: none;
 }
-
 /* Footer bar styling */
 footer{
```
Comment (haefnir) [inline stylesheet.css:174]: footer breaks at resolutions with width below 365px, consider adding a conditional rule to the media check

File: stylesheet.css [hunk 29]
```diff
@@ -105,186 +99,207 @@ button{
...
-        display: grid;
         grid-template-columns: 100%;
-        margin:auto;
-        padding:10px;
+        margin: 0;
+        padding: 10px;
     }
-    
     .contact_form{
         background-color:#FFFFFF;
```
Comment (haefnir) [inline stylesheet.css]: this is defined earlier in the stylesheet, so isnt needed

File: stylesheet.css [hunk 30]
```diff
@@ -105,186 +99,207 @@ button{
...
-        padding:0;
+        padding: 0;
```
Comment (haefnir) [inline stylesheet.css]: this padding is being overriden in this block. delete

File: stylesheet.css [hunk 31]
```diff
@@ -105,186 +99,207 @@ button{
...
-        width:100%
+        /* margin: auto; */
+        width: 80%;
     }
 
+}
+@media only screen and (max-width: 617px){
+    nav a{
+        display: grid;
+		grid-template-columns: 100%;
```
Comment (haefnir) [inline stylesheet.css:271]: fix indentation

**Review by haefnir — CHANGES_REQUESTED** [review]

**Review by haefnir — APPROVED** [review]
Footer works, approved!

**Pull request merged**